### DISCUSSION TITLE
Specify that GLOP and CLP should use primal simplex?

### DISCUSSION THREAD

**Discussion by gdowdy3** [google/or-tools#4003] [Linear Solver questions]
**What language and solver does this apply to?**
Python, GLOP, and CLP

**Describe the problem you are trying to solve.**
I would like to be able to use GLOP/CLP to solve an LP and specify that the solver should use the primal simplex algorithm.  I suspect that this can be done using the `SetSolverSpecificParametersAsString` method, but I don't know what argument to pass to this method.  Can it be done?

**Comment by lperron** [comment]
1. Which library ? MPSolver ? ModelBuilder ? MathOpt ?
2. I do not know the parameter structure for CLP
3. for GLOP: use_dual_simplex: false
4. MPSolver supports basic parameters without using the string version.
